src/nodejs/rpc-over-https/v2.9.0/client/module/getNodesBy.js

```diff
@@ -101,9 +98,9 @@ exports.ping = (options) => {
   let nodes = options.nodes
 
   if (defly) {
-    dbg.logDeep(__filename + ': getNodesByPing().options: ', opts)
-    dbg.logDeep(__filename + ': getNodesByPing().net: ', net)
-    dbg.logDeep(__filename + ': getNodesByPing().cfg.GetNodes(): ', cfgNodes)
+    // dbg.logDeep(__filename + ': getNodesByPing().options: ', opts)
```

Comment:
these 2 lines can be removed?

Reply:
I'm keeping for now as I'm not sure I want to remove. They may be relevant again. If not, I'll remove permanently. Thanks!